dbt/include/teradata/macros/materializations/incremental/helpers.sql

```diff
@@ -25,6 +25,6 @@
   {% elif strategy == 'merge' %}
     {% do return(teradata__get_merge_sql(target_relation, tmp_relation, unique_key, dest_columns,incremental_predicates)) %}
   {% else %}
-    {% do exceptions.CompilationError("Invalid Strategy") %}
+    {% do exceptions.raise_compiler_error("Invalid Strategy") %}

```

Comment:
@septimit, is this change suggested by the dbt-core?
Can you please point us to any document for the same>

Reply:
please see https://github.com/search?q=repo%3Adbt-labs%2Fdbt-core+exceptions.raise_compiler_error&type=code or https://github.com/search?q=repo%3Adbt-labs%2Fdbt-core+exceptions.CompilationError&type=code